utils/install_tools.sh

```diff
@@ -34,13 +34,13 @@ install_native ()
 		sudo dpkg -i advance-toolchain-at*devel_* \
 			advance-toolchain-at*mcore-libs_* \
 			advance-toolchain-at*perf_* || return ${?}
+		if [ -e advance-toolchain-*libnxz* ]; then
```

Comment:
This matches more than one file, so the test fails, and the libnxz package is not installed. That's why when installing the dbg packages it fails about libnxz-dbg package.

So I'd put this : `-e advance-toolchain-at*libnxz_*`